controllers/ptpoperatorconfig_controller.go

```
@@ -465,15 +469,15 @@ func (r *PtpOperatorConfigReconciler) SetupWithManager(mgr ctrl.Manager) error {
-		return DefaultTransportHost, nil
+			transportHost, DefaultTransportHost())
+		return DefaultTransportHost(), nil
 	}
 
 	_, err := url.Parse(transportHost)
 	if err != nil {
 		glog.Warningf("ptp operator config Spec, ptpEventConfig.transportHost=%v is not valid, proceed as %s",
-			transportHost, DefaultTransportHost)
-		return DefaultTransportHost, nil
+			transportHost, DefaultTransportHost())
+		return DefaultTransportHost(), nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 476) but it returns nil

(nilerr)